lib/puppet/util/environment_mode.rb

```diff
@@ -0,0 +1,23 @@
+module Puppet
+  module Util
+    class EnvironmentMode
+      def initialize(name)
+        @name = name.to_sym
+      end
+
+      attr :name
+
+      def local?
```

Comment:
If we're not going to use the methods, having a class rather than several symbols seems heavy-handed.